### ISSUE TITLE
TOPO correction returns noise

### ISSUE THREAD

**Issue by jantin1** [EnSpec/hytools#12]
I am trying to correct an aerial hyperspectral image topographically. I have a DEM acquired in the same time as the image and with the same spatial resolution and coverage. I have created slope and aspect maps and calculated solar angles and fed the whole package to hyTools. 
I used modified scripts from this github to create a config .json and to perform the correction. 
Nevertheless the result of the processing is colorful noise, which looks like it has nothing to do with the HS image or the DEM. 
What do I do wrong  to get this kind of result? Is there some setting or input I forget or don't know about? Attached is the image of my product. I work on PopOS flavour of Ubuntu. 
![image](https://user-images.githubusercontent.com/91844627/176761130-fc922313-0f40-4cfd-a53f-91dd9c14e93d.png)

**Comment by adamchlus** [comment]
It’s difficult to diagnose the issue from the screenshot alone, but it looks as though there is an issue with the interleaving of the dataset(s). I would recommend double-checking the data by visually inspecting all of the input images as well as ensuring the ancillary datasets are in the correct units, see below (ignore unused bands):

           1) path length (sensor-to-ground in meters)
           2) to-sensor-azimuth (0 to 360 degrees clockwise from N)
           3) to-sensor-zenith (0 to 90 degrees from zenith)
           4) to-sun-azimuth (0 to 360 degrees clockwise from N)
           5) to-sun-zenith (0 to 90 degrees from zenith)
           6) solar phase (degrees between to-sensor and to-sun vectors in principal plane)
           7) slope (local surface slope as derived from DEM in degrees)
           8) aspect (local surface aspect 0 to 360 degrees clockwise from N)
           9) cosine i (apparent local illumination factor based on DEM slope and aspect

**Comment by jantin1** [comment]
Thank you for the reply. My ancillaries (terrain slope and aspect, solar azimuth and zenith) are all in radians as this is what your scripts require. Should they be recalculated to degrees?

**Comment by adamchlus** [comment]
Yes, the angles in the ancillary files should in degrees, however any angles in the config file should be specified in radians.
